app.py

```diff
@@ -1,80 +1,90 @@
-'''
+"""
```

Comment:
nit: It might be better to not update the code irrelevant to the bug in the same PR. This makes it the PR look simpler to review and easier to catch bugs. This might be better done in a separate PR